.circleci/config.yml

```diff
@@ -166,7 +162,7 @@ jobs:
           name: Compile
           command: yarn compile
       - run:
-          name: Publish all packages
+          name: Publish all packages # TODO can we exclude service-utils?
```

Comment:
@zachkirsch is there an easy way to do this? https://github.com/yarnpkg/yarn/issues/7051